src/services/device.service.ts

```diff
@@ -0,0 +1,31 @@
+import { db } from '../database';
+
+export interface Device {
+  userId: number;
+  deviceToken: string;
+}
+
+interface InsertDeviceResult {
+  userId: number;
+  deviceToken: string;
```

Comment:
This interface is supposed to represent what the database returns, and the database uses snake_case rather than camelCase.

```suggestion
interface InsertDeviceResult {
  user_id: number;
  device_token: string;
```